index.bs

```diff
@@ -119,6 +119,19 @@ To <dfn>Construct an AmbientLightSensor Object</dfn> the user agent must invoke
 The <a attribute for="AmbientLightSensor">illuminance</a> attribute of the {{AmbientLightSensor}}
 interface represents the <a>current light level</a>.
 
+Use Cases and Requirements {#usecases-requirements}
+=========
+
+1. A Web application gradually updates document style based on light level changes.
+1. A Web application provides input for a smart home system to control lighting.
+1. A Web aplication checks whether light level at work space is sufficient, based on occupational
```

Comment:
My question was whether onboard sensors were good enough for meaningfully measuring that. My hunch is that onboard sensors are good enough to measure change, but probably not absolute values. I might be completely wrong, however.

Overall, this is a cool use case, I'm just concerned that it is not doable in practice because of HW constraints.